ansible/roles/neutron/handlers/main.yml

```diff
@@ -104,6 +150,11 @@
     healthcheck: "{{ service.healthcheck | default(omit) }}"
   when:
     - kolla_action != "config"
+    - inventory_hostname == item
+  loop: "{{ groups['neutron_l3_agent_running_True'] | default([]) }}"
```

Comment:
Unfortunately this pattern no longer works with the version of Ansible used in Antelope. We can keep it in Zed though.

Reply:
That's a shame. One to look out for in the future then.